app/upgrades/v1/constants.go

```diff
@@ -0,0 +1,23 @@
+package v1
+
+import (
+	"github.com/eve-network/eve/app/upgrades"
+	feemarkettypes "github.com/skip-mev/feemarket/x/feemarket/types"
+
+	store "cosmossdk.io/store/types"
+)
+
+const (
+	// UpgradeName defines the on-chain upgrade name.
+	UpgradeName = "v1"
```

Comment:
looks good please change it to v0.0.6, also change folder name

Reply:
also we need add v0.0.6 upgrade to here -> https://github.com/eve-network/eve/blob/main/app/upgrades.go#L28

Reply:
I think v0.1.0 is more suitable since we are adding a new module

Reply:
let do it, thank you